modules/installation/proc-rhdh-deploy-eks-operator.adoc

```diff
@@ -1,12 +1,17 @@
-[id='proc-rhdh-deploy-eks-using-operator_{context}']
+// Module included in the following assemblies
+// assembly-install-rhdh-eks.adoc
 
-= Deploying {product} on {eks-name} ({eks-short}) using the Operator
+[id='proc-rhdh-deploy-eks-operator_{context}']
+= Installing {product-short} on {eks-short} with the Operator
 
-You can deploy the {product-short} on {eks-short} using the {product} Operator with or without https://olm.operatorframework.io[Operator Lifecycle Manager (OLM) framework]. Following that, you can proceed to install your {product-short} instance in {eks-short}.
+You can install the {product} Operator with or without the Operator Lifecycle Manager (OLM) framework.
```

Comment:
[SME] it would be nice to provide some more conceptual info to help the customer understand whether they should install with or without the OLM framework - can we talk about what add'l info we should include here?

Reply:
Maybe we should include a link to https://olm.operatorframework.io/docs/ ?